packages/cli/test/integration/users.endpoints.test.ts

```diff
@@ -46,25 +47,17 @@ beforeAll(async () => {
 
 	utils.initTestTelemetry();
 	utils.initTestLogger();
+
+	jest.setTimeout(10000); // fake SMTP service might be slow
```

Comment:
does this overwrite the `jest.setTimeout(30000)` for mysql?

Reply:
They should be separate timeouts. The timeout for MySQL in `setup.ts` (non-test file) is 30 seconds and the timeout in these two suites before the change was 5 seconds, which is why the error Jan showed said 5 seconds, not 30.

Reply:
yes, but I guess jan tested it with sqlite where we don't set a special timeout, but for mysql we do. Just want to make sure that it doesn't overwrite the one from mysql